.prettierrc.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
   jsxBracketSameLine: true,
   trailingComma: 'es5',
   printWidth: 80,
-  parser: 'babel',
-
+  parser: 'flow',
```

Comment:
should this be 'hermes' or 'flow'?

Reply:
The hermes PR wasn't merged yet due to concerns of size IIRC. Looks like flow is current enough for the moment to parser everything. (not actually sure what the plan is going forward with hermes vs. flow parser).

Reply:
Ah I see